framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Widgets/TransactionWidget.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import { TableBody, TableHeader } from '../Table';
+import { Widget, WidgetHeader, WidgetBody } from '../widget';
+import Text from '../Text';
+import Grid from '../Grid';
+import CopiableText from '../CopiableText';
+import styles from './Widgets.module.scss';
+
+interface WidgetProps {
+	scrollbar?: boolean;
+	transactions: Record<string, string>[];
+	widgetTitle: string;
+}
+
+const TransactionWidget: React.FC<WidgetProps> = props => {
+	if (props.transactions.length === 0) return null;
```

Comment:
component shouldnt return `null` value. it should at least return `<div />`.
but why when length is zero, it's trying to not to display anything?